python_backend/langchain_pipeline/retriever/vector_db.py

```
@@ -137,61 +193,151 @@ def save_vector_store(vectorstore: FAISS, save_path: Path):
+        except Exception as security_error:
+            logger.warning(f"안전한 역직렬화 실패, 신뢰된 경로 재시도: {security_error}")
+            # 신뢰된 경로에서만 위험한 역직렬화 허용
+            if _is_project_managed_path(load_path):
+                vectorstore = FAISS.load_local(
+                    str(load_path),
+                    embeddings,
+                    allow_dangerous_deserialization=True
+                )
+                logger.warning("프로젝트 관리 경로에서 위험한 역직렬화 사용됨")
+            else:
+                raise security_error
```

Ruff (0.13.1)

Use `raise` without specifying exception name

Remove exception name

(TRY201)